wrappers/services.go

```diff
@@ -1039,7 +1217,7 @@ func RestartServices(apps []*snap.AppInfo, explicitServices []string,
 // together with their enable/disable status.
 func ServicesEnableState(s *snap.Info, inter Interacter) (map[string]bool, error) {
 	sysd := systemd.New(systemd.SystemMode, inter)
-	sts, err := internal.QueryServiceStatusMany(sysd, s.Services())
+	sts, _, err := internal.QueryServiceStatusMany(s.Services(), sysd)
```

Comment:
we probably want to add a TODO and discuss how to define the result as a service could be enabled for some users and not others